www/docs/src/content/docs/how-it-works/index.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: "How does it work!?"
+description: "How does Astro StudioCMS Work?  Well here is all the answers!"
+---
+
+# Introduction
+
+StudioCMS is a CMS built for Astro. It is a powerful tool that allows you to create, edit, and delete content for your Astro project in an SSR deployment while connected to [Astro Studio](https://studio.astro.build). It is designed to be easy to use and flexible enough to handle any type of content.
+
+Extending StudioCMS is easy too! You can also create your own plugins to add new features to StudioCMS.
+
+## How does it work?
+
+StudioCMS is divided into multiple parts:
+
+- **StudioCMS Core**: The core of StudioCMS is the primary integration that hooks into your Astro project and provides the ability to interact with the Astro Studio (through AstroDB) integration with a prebuilt front-end.
+- **StudioCMS: Dashboard**: The StudioCMS dashboard is the primary web interface for managing your AstroStudioCMS project. It provides a user-friendly interface for creating, editing, and deleting content for your Astro project.
+- **StudioCMS: ImageHandler**: This integration provides a `<CustomImage />` Astro component that is used to handle remote images and optimize them for your Astro project in a SSR environment.
+- **StudioCMS: Robots-TXT**: This integration automates the creation of a `robots.txt` file for your Astro project and AstroStudioCMS.
+
+## Breakdown
+
+### StudioCMS Core
+
+#### Overview
+
+StudioCMS Core is an Astro Integration which provides a comprehensive CMS solution out-of-the-box. This integration registers StudioCMS as the first plugin, sets up database configurations, watches for configuration changes, and adds various functionalities including virtual imports, logging, and default frontend routes.
+
+#### Features
+- **@astrojs/db**: The StudioCMS Core integration uses the `@astrojs/db` integration to connect to AstroStudio.
+- **Virtual Imports**: The StudioCMS Core integration adds several [Vite Virtual Imports](https://vitejs.dev/guide/api-plugin#virtual-modules-convention) packed with features that give you full control over your dynamic content.
+- **Logging**: The StudioCMS Core integration logs all changes to Astro Studio and can be enabled or disabled through a config option.
+- **Default Frontend Routes**: The StudioCMS Core integration adds default frontend routes to your Astro project to allow for easy access to the StudioCMS dashboard.
+- **Plugin System**: You can easily extend StudioCMS using its own plugin system to create additional functionality.
```

Comment:
I'm only going through this page at the moment, so I'm not sure, but do we have a config reference we can link to to disable logging? I also removed "enabled" as I'm guessing it's enabled by default? If not, replace disabled with enabled.

By "default frontend routes", if we mean the dashboard, we should probably just say dashboard. I haven't changed this in the suggestion as the language is also used elsewhere.
```suggestion
#### Features

- **@astrojs/db**: StudioCMS Core uses the `@astrojs/db` integration to connect to Astro Studio.
- **Virtual Imports**: StudioCMS Core adds several [virtual modules](https://vitejs.dev/guide/api-plugin#virtual-modules-convention) packed with features that give you full control over your dynamic content.
- **Logging**: StudioCMS Core logs all changes to Astro Studio. This can be disabled in your config file.
- **Default Frontend Routes**: The StudioCMS Core integration adds default frontend routes to your Astro project to allow for easy access to the StudioCMS dashboard.
- **Plugin System**: You can extend StudioCMS using plugins to add additional functionality.
```

Reply:
Logging is disabled by default, the "default front end routes"  are the `/` and `/[slug]` routes, the dashboard routes are handled separately by the actual Dashboard integration

Reply:
Updated the suggestion